src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_prompt/index.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Fragment } from 'react';
+import {
+  EuiEmptyPrompt,
+  EuiIcon,
+  EuiText,
+  EuiHorizontalRule,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { CreateButton } from '../create_button';
+
+export const CreateIndexPatternPrompt = ({
+  indexPatternCreationOptions
+}) => (
+  <EuiEmptyPrompt
+    className="euiPanel"
+    iconType="indexPatternApp"
+    title={
+      <EuiText grow={false}>
+        <h2>
+          <FormattedMessage id="kbn.management.indexPatternPrompt.title" defaultMessage="Create your first index pattern" />
+        </h2>
+      </EuiText>}
+    body={
+      <Fragment>
+        <p style={{ padding: '0 24px' }}>
+          <FormattedMessage
+            id="kbn.management.indexPatternPrompt.subtitle"
+            defaultMessage="Index patterns allow you to bucket disparate data sources together so their shared fields may be queried in
+              Kibana."
+          />
+        </p>
+        <EuiHorizontalRule margin="m" />
+        <p style={{ textAlign: 'left' }}>
+          <FormattedMessage
+            id="kbn.management.indexPatternPrompt.examplesTitle"
+            defaultMessage="Examples of index patterns"
+          />
+        </p>
+        <div style={{ textAlign: 'left' }}>
```

Comment:
Instead of using a div with inline styles and wrapping EUI flex elements, use either an `EuiDescriptionList` (https://elastic.github.io/eui/#/display/description-list) or an `EuiListGroup` (https://elastic.github.io/eui/#/display/list-group). 

This would provide a more consistent design with rest of Kibana and allow you to remove some styles here: https://github.com/elastic/kibana/blob/4411135421001e967022923999f9fe89ecc0d0fc/src/legacy/core_plugins/kibana/public/management/sections/index_patterns/index.scss#L16-L18

Reply:
Addressed this in https://github.com/mattkime/kibana/pull/4